airflow/config_templates/default_airflow.cfg

```
@@ -776,7 +776,9 @@ tolerations =
 # List of supported params in **kwargs are similar for all core_v1_apis, hence a single config variable for all apis
 # See:
 #   https://raw.githubusercontent.com/kubernetes-client/python/master/kubernetes/client/apis/core_v1_api.py
-kube_client_request_args =
+# Note that if no _request_timeout is specified, the kubernetes client will wait indefinitely for kubernetes
+# api responses, which will cause the scheduler to hang. The timeout is specified as [connect timeout, read timeout]
+kube_client_request_args = {"_request_timeout" : [60,60] }
```

```suggestion
kube_client_request_args = {{"_request_timeout" : [60,60] }}
```

The problem is this file isn't a plain config file, it's actually a giant python format string.

---

Setting this in the `~/airflow/airflow.cfg` wouldn't have the same problem, it's just this default config template that is the problem.